src/backend/aspen/workflows/nextstrain_run/build_config.py

```diff
@@ -29,7 +29,9 @@ def update_subsampling(self, config, subsampling):
         if self.group.name == "Chicago Department of Public Health":
             subsampling["group"][
                 "query"
-            ] = "--query \"((location == '{location}') & (division == '{division}')) | submitting_lab == 'RIPHL at Rush University Medical Center'\""
+            ] = '''--query "((location == '{location}') & (division == '{division}')) | submitting_lab == 'RIPHL at Rush University Medical Center'"'''
```

Comment:
Most of our subsampling queries have *both* literal ' and " characters in them, and we *also* need a way to tell python where to start and stop reading the string. If we use " or ' to wrap the outside of the string, we need to escape (`\"` or `\'`) those characters inside the string. This is fine/normal, but it makes our strings look different than the ones we copy out of your YAML files. Using triple quotes (or triple double quotes) lets us use those characters more naturally in our script, they won't need to be escaped since Python is only looking for `'''` to mark the end fo the string.
https://towardsdatascience.com/single-double-and-triple-quotes-in-python-7ceea990baf